gulpfile.js

```diff
@@ -219,6 +219,16 @@ const mappings = {
     'dir': 'powerbiembedded/lib',
     'source': 'arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json'
   },
+  'rdbms.mysql': {
+    'packageName': 'azure-arm-rdbms',
+    'dir': 'rdbmsManagement/lib/mysql',
+    'source': 'arm-rdbms/2017-04-30-preview/swagger/Rdms.MySQL.json'
+  },
```

Comment:
I think this file (gulp task) will execute on `azure-rest-api-specs` repo. So, if someone (like my team) wants to generate all sdks to publish a refresh (we do language updates for eg.), this script will break because this spec is not in the public repo. I think the right thing to do here is to move your spec to the public repo

---
In reply to: [118066968](https://github.com/Azure/azure-sdk-for-node/pull/2173#discussion_r118066968) [](ancestors = 118066968)